instrumentation/graphql-java/graphql-java-12.0/library/build.gradle.kts

```diff
@@ -0,0 +1,10 @@
+plugins {
+  id("otel.library-instrumentation")
+}
+
+dependencies {
+  library("com.graphql-java:graphql-java:12.0")
+  implementation(project(":instrumentation:graphql-java:graphql-java-common:library"))
+
+  testImplementation(project(":instrumentation:graphql-java:graphql-java-common:testing"))
```

Comment:
do we need `latestDepTestLibrary`, or that is not needed until next graphql release when the deprecated methods are removed?

Reply:
I'll add it. Yes, currently it isn't needed because the tests pass with the latest version. I think this is the first time we have tried to fix the instrumentation before it actually breaks.